backend/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,70 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.224.2/containers/dotnet
+{
+	"name": "C# (.NET)",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": {
+			// Update 'VARIANT' to pick a .NET Core version: 3.1, 5.0, 6.0
+			// Append -bullseye or -focal to pin to an OS version.
+			"VARIANT": "6.0",
```

Comment:
Should the "VARIANT" be .NET Core version 5.0 here as well?

Reply:
I tried this first of all.  Found out that it wouldn't work for some reason.  It's backward compatible though.